LoRaEngine/deployment.test.template.json

```diff
@@ -143,7 +143,7 @@
           "sensordecodermodule": {
             "type": "docker",
             "settings": {
-              "image": "$CONTAINER_REGISTRY_ADDRESS/decodersample:2.0",
+              "image": "$CONTAINER_REGISTRY_ADDRESS/decodersample:2.0-arm32v7",
```

Comment:
Is it OK to hard-code the version 2.0 here?

Reply:
I will remove arm32